shared/src/commonMain/kotlin/org/hyperskill/app/study_plan/screen/view/StudyPlanScreenViewState.kt

```diff
@@ -1,11 +1,13 @@
 package org.hyperskill.app.study_plan.screen.view
 
 import org.hyperskill.app.gamification_toolbar.presentation.GamificationToolbarFeature
+import org.hyperskill.app.problems_limit.presentation.ProblemsLimitFeature
 import org.hyperskill.app.study_plan.widget.view.StudyPlanWidgetViewState
 
 data class StudyPlanScreenViewState(
```

Comment:
I think we should move `StudyPlanScreenViewState` to the scope of the `StudyPlanScreenFeature` -> `StudyPlanScreenFeature.ViewState`

cc @XanderZhu

Reply:
The only one view state that has separate file except this - `StudyPlanWidgetViewState`, but it seems really complex to hold it separately, I agree with this point